api/src/main/resources/application-test.yml

```diff
@@ -0,0 +1,11 @@
+server:
+  port: 8081
+
+application:
+  admin:
+    password: "{bcrypt}$2a$10$jXI7IkpEVqv7Qq.UfQUhO.353gfrtL/qxPFHLZMinUFw5KObS54mi"
+  datasource:
```

Comment:
do bazki powinny być propertasy
```
spring:
  datasource:
```
Application to nasz własny przedrostek. Driver-class-name niepotrzebny, bo to sobie ogarnie Spring z zależności, to więc wywalić